benchmark/torch/ppo/README.md

```diff
@@ -21,11 +22,11 @@ Please see [mujoco-py](https://github.com/openai/mujoco-py) to know more about M
 
 ## How to use
 ### Dependencies:
-+ python>=3.6.2
++ python>=3.7
 + [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
 + pytorch
-+ gym==0.18.0
-+ mujoco-py==2.1.2.14
++ gym>=0.26.0
++ mujoco>=2.2.2
 + atari-py==0.2.6
 
 ### Training:
```

Comment:
Please update the default environment name of `benchmark/torch/ppo/mujoco_config.py`